CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to {{ project }}
+
+The {{ project }} project team welcomes contributions from the community. Before you start working with {{ project }}, please
+read our [Developer Certificate of Origin](https://cla.vmware.com/dco). All contributions to this repository must be
+signed as described on that page. Your signature certifies that you wrote the patch or have the right to pass it on
+as an open-source patch.
+
```

Comment:
I think this is missing a section about other ways to contribute. Maybe a stripped down version of something like this: https://github.com/cncf/project-template/blob/main/CONTRIBUTING.md#ways-to-contribute